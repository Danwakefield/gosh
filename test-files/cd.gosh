echo "3 test cases"
A=$(pwd)
if [ $A = $PWD ]; then
    echo "SUCCESS 1"
else
    echo "FAIL 1"
fi

cd /
A=$(pwd)
if [ $A = $PWD ]; then
    echo "SUCCESS 2"
else
    echo "FAIL 2"
fi

cd
A=$(pwd)
if [ $A = $PWD ]; then
    echo "SUCCESS 3"
else
    echo "FAIL 3"
fi
