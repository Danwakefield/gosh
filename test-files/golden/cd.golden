3 test cases
SUCCESS 1
SUCCESS 2
SUCCESS 3
